L1Trigger/L1TMuonEndCap/src/PtAssignmentEngine2017.cc

```diff
@@ -1,6 +1,6 @@
 #include "L1Trigger/L1TMuonEndCap/interface/PtAssignmentEngine2017.h"
 #include "L1Trigger/L1TMuonEndCap/interface/PtAssignmentEngineAux2017.h"
-#include "L1Trigger/L1TMuonEndCap/interface/PtLutVarCalc.h"
+#include "L1Trigger/L1TMuonEndCap/interface/PtLUTVarCalc.h"
```

Comment:
Other than this line, no changes to this file except spaces ... will probably make the review easier if we just leave it untouched.  Also, the central CMSSW folks prefer to keep file names the same unless there's a strong motivation to change (right, @rekovic )?  So I'd suggest leaving PtLutVarCalc.

Reply:
Khristian calls it PtLut, but everywhere inside L1Trigger/L1TMuonEndCap it is called PtLUT. Also, this does not affect anything outside of L1Trigger/L1TMuonEndCap .

Reply:
I guess this makes it consistent with PtLUTReader.h and PtLUTWriter.h, which is good.

Reply:
Also, the variable ptLUTVersion_ and the class EMTFPtLUT , just to motivate it more